src/requires/spglib.jl

```diff
@@ -0,0 +1,46 @@
+# Functionalities that are available when Spglib is loaded
+
+using Bravais: reciprocalbasis, DirectBasis
+using LinearAlgebra
+import ..KPaths: irrfbz_path
+
+export irrfbz_path
+
+"""
+    irrfbz_path(cell::Spglib.Cell)  -->  ::KPath{D}
+
+Returns a k-path for arbitrary 3D cell in the Spglib format. The k-path is equivalent to the
+"standard" k-path given by `irrfbz_path` for the equivalent standard primitive lattice.
+Does not give the correct k-path if the input cell is a supercell of a smaller primitive
+cell (a warning is printed).
+"""
+function irrfbz_path(cell::Spglib.Cell)
+    # standardize cell
+    dset = Spglib.get_dataset(cell)
+    sgnum = dset.spacegroup_number
+    std_lattice = DirectBasis(collect(eachcol(dset.std_lattice)))
```

Comment:
```suggestion
    std_lattice = DirectBasis{3}(collect(eachcol(dset.std_lattice)))
```
This is only going to work for 3D systems anyway, right? (otherwise, this is infers as `Any`)